packages/web/src/components/search/SearchBox.js

```diff
@@ -1665,7 +1687,7 @@ const SearchBox = (props) => {
 											ref={_dropdownULRef}
 											className={`${getClassName(props.innerClass, 'list')}`}
 										>
-											{showAIScreen && (
+											{showAIScreen || isTestingAI ? (
```

Comment:
@SavvyShah Can't we set `showAIScreen ` to true for test mode instead of using `isTestingAI` var?

Reply:
OK